Frontend/ui-driver/src/Screens/Types.purs

```diff
@@ -954,7 +955,8 @@ type TripDetailsScreenData =
     totalAmount :: Int,
     paymentMode :: PaymentMode,
     distance :: String,
-    status :: String
+    status :: String,
+    vehicleType :: String
```

Comment:
@Anuragini-2506 can we create a data for this.

Reply:
done

Reply:
Will do this changes and raise a separate PR for refactoring the vehicleType in driver and customer side